packages/opentelemetry/lib/tracing/tracerProvider.js

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable no-unused-vars */
+
+import { getBatchSpanProcessor } from './spanProcessorFactory';
+import { buildExporter, AVAILABLE_EXPORTERS } from './exporterFactory';
+import { NodeTracerProvider } from '@opentelemetry/node';
+import { registerInstrumentations, InstrumentationOption } from '@opentelemetry/instrumentation';
+import { ServerInstrumentation } from './serverInstrumenation';
+import { diag, DiagConsoleLogger, DiagLogLevel } from '@opentelemetry/api';
+import { SpanProcessor, SpanExporter } from '@opentelemetry/tracing';
+import { ExporterConfig } from '@opentelemetry/exporter-jaeger';
+import _ from 'lodash';
+import { forEachPromise } from '../utils';
+
+//Delegate
+class TracerProvider {
+  constructor () {
+    this.provider = new NodeTracerProvider();
+    this.exporterInstances = {};
+    this._currentConfig = {
+      active: true,
+      exporterConfig: []
```

Comment:
exporterConfig or exporterConfigs?

Reply:
yeah i thought of that, for some reason it didnt sound right :P , but since you another affirmation will update :)